v2/datastream-to-spanner/terraform/samples/mysql-end-to-end/terraform.tfvars

```diff
@@ -52,18 +55,22 @@ dataflow_params = {
     dlq_retry_minutes                   = 10                                             # Adjust as needed
     dlq_max_retry_count                 = 3                                              # Adjust as needed
     datastream_root_url                 = "<YOUR_DATASTREAM_ROOT_URL>"                   # Base URL of your Datastream API (optional)
-    datastream_source_type              = "MYSQL"
+    datastream_source_type              = "mysql"
     round_json_decimals                 = false
-    run_mode                            = "STREAMING"
+    run_mode                            = "regular"
     transformation_context_file_path    = "<YOUR_TRANSFORMATION_FILE_PATH>" # Path to your transformation file (optional)
     directory_watch_duration_in_minutes = "5"                               # Adjust as needed
-    spanner_priority                    = "high"
-    dlq_gcs_pub_sub_subscription        = "<YOUR_DLQ_PUBSUB_SUBSCRIPTION>" # Optional
+    spanner_priority                    = "HIGH"
+    dlq_gcs_pub_sub_subscription        = "<YOUR_DLQ_PUBSUB_SUBSCRIPTION>"        # Optional
+    transformation_jar_path             = "<YOUR_CUSTOM_TRANSFORMATION_JAR_PATH>" # Optional
+    transformation_custom_parameters    = "<YOUR_CUSTOM_PARAMETERS_FOR_JAR>"      # Optional
+    transformation_class_name           = "<YOUR_TRANSFORMATION_CLASS_NAME>"      # Fully Classified Class Name(Optional)
+    filtered_events_directory           = "<YOUR_GCS_PATH_FOR_FILTERED_EVENTS>"   # Optional
   }
 
   runner_params = {
     additional_experiments       = []                           # Add any additional experiments or leave empty
-    autoscaling_algorithm        = "THROUGHPUT_BASED"           # Or NONE
+    autoscaling_algorithm        = "BASIC"                      # Or NONE
```

Comment:
Change this back I think?

https://cloud.google.com/dataflow/docs/reference/pipeline-options